history.html

```diff
@@ -536,9 +536,9 @@ <h2 id="history">
         <address>
           Anssi Kostiainen, Chair<br>
           <span class="vcard"><a class="url fn" rel="author" href=
-          "https://www.w3.org/People/Dom/">Dominique , Staff
-          Contacts</a></span><br>
-          Haza&#235;l-Massieux, Fuqiao Xue, Staff Contacts<br>
+          "https://www.w3.org/People/Dom/">Dominique Haza&#235;l-Massieux, Staff
+          Contact</a></span><br>
+          Fuqiao Xue, Staff Contact<br>
         </address>
```

Comment:
maybe instead `Dominique Hazael-Massieux, Fuqai Xue, Staff Contacts` to avoid the repetition?

Reply:
Done. Thank you!